src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -3250,7 +3248,8 @@ public void T111_String_Create_Write_Star()
         {
             var tempGuidName = GetTempFileInfoWithExtension(".docx");
             // opening the package attempts to read the package, and no permissions.
```

Comment:
is this comment correct?

Reply:
i honestly didn't even really know what it means, mainly the "and no permissions" part

Reply:
Yep, maybe just delete it then, since none of this looks correct.